vignettes/GenomicFiles.Rmd

```diff
@@ -0,0 +1,691 @@
+---
+title: "Introduction to GenomicFiles"
+authors:
+  - name: "Valerie Obenchain"
+  - name: "Michael Love"
+  - name: "Shepherd"
+  - name: "Martin Morgan"
```

Comment:
In addition to you, the only authors that should be listed here should be the ones in the PDF: https://bioconductor.org/packages/3.17/bioc/vignettes/GenomicFiles/inst/doc/GenomicFiles.pdf